lib/_validate.js

```diff
@@ -182,6 +182,30 @@ const validations = {
       )
     }
 
+    return true
+  },
+  debug: (value, context) => {
```

Comment:
can't tell if this function ever returns false or not

Reply:
It either returns an error or true

Reply:
seems strange 
- could it return a true / false ?

Reply:
It already returns true if there is no error and if not, it throws an error

Reply:
`return true` is just a notation. It can easily `return` for fast-fail. I don't think the return value is checked anywhere.

Reply:
yes we've reached the same conclusion.. we are leaving it like it is now for consistency sake